Makefile

```diff
@@ -60,7 +60,7 @@ html_coverage: ## Generate HTML coverage report
 	${VENV_BIN}/coverage html
 
 quality: ## Run quality checks
-	${VENV_BIN}/pycodestyle --config=pycodestyle blockstore *.py
-	${VENV_BIN}/pylint --rcfile=pylintrc blockstore *.py
+	${VENV_BIN}/pycodestyle --config=pycodestyle blockstore tagstore *.py
+	${VENV_BIN}/pylint --rcfile=pylintrc blockstore tagstore *.py
```

Comment:
If you also update `.coveragerc`, you can see the coverage missing from the `tagstore` code.  
E.g, if you use this:
```diff
diff --git a/.coveragerc b/.coveragerc
index 4c4c847..5f76dff 100644
--- a/.coveragerc
+++ b/.coveragerc
@@ -1,7 +1,9 @@
 [run]
 branch = True
 data_file = .coverage
-source=blockstore
+source=
+    blockstore
+    tagstore
 omit =
     blockstore/settings*
     blockstore/conf*
@@ -10,3 +12,7 @@ omit =
     *admin.py
     *static*
     *templates*
+[report]
+exclude_lines =
+    raise NotImplementedError()
+    yield None
```

then the coverage report for this change is:
```
tagstore/__init__.py (100%)
tagstore/backends/django.py (95.2%): Missing lines 36-37,105,126
tagstore/backends/tagstore_django/models.py (100%)
tagstore/backends/tests.py (100%)
tagstore/models/__init__.py (100%)
tagstore/models/entity.py (100%)
tagstore/models/tag.py (100%)
tagstore/models/taxonomy.py (100%)
tagstore/models/user.py (100%)
tagstore/tagstore.py (85.0%): Missing lines 53,57,91-94
-------------
Total:   372 lines
Missing: 10 lines
Coverage: 97%
```

Reply:
Thanks, updated.